compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -805,11 +836,13 @@ static LogicalResult setWarpReductionConfig(func::FuncOp entryPoint,
   if (numDynamicReductionDims) {
     SmallVector<int64_t> reductionTileSizes(op.getNumLoops(), 0);
     // TODO: Don't hard code this.
-    reductionTileSizes[reductionDims[0]] = kCudaWarpSize;
+    reductionTileSizes[reductionDims[0]] =
```

Comment:
Nit: I'd typically create a local variable for long expressions like `targetInfo.supportedSubgroupSizes` given it's used multiple times. Makes the code a bit nicer to read.